atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleQueryFactory.java

```diff
@@ -317,6 +323,12 @@ private ArrayHandler cellsAndTimestampsToOracleArray(Collection<Map.Entry<Cell,
             Cell cell = entry.getKey();
             oraRows.add(new Object[] { cell.getRowName(), cell.getColumnName(), entry.getValue() });
         }
-        return jdbcHandler.createStructArray("PT_MET_CELL_TS", "PT_MET_CELL_TS_TABLE", oraRows);
+        return config.jdbcHandler().createStructArray(
+                config.shared().tablePrefix() + "CELL_TS",
```

Comment:
This construct also shows up quite a bit, might be worth having a util method as well.